src/DIRAC/DataManagementSystem/Client/FileCatalogClientCLI.py

```diff
@@ -584,7 +579,7 @@ def registerFile(self, args):
         infoDict["Size"] = int(args[2])
         infoDict["SE"] = args[3]
         if args[4] == "None":
-            _status, guid = commands.getstatusoutput("uuidgen")
+            _status, guid = subprocess.getstatusoutput("uuidgen")
```

Comment:
You'll also need to import `uuid`:
```suggestion
            guid = str(uuid.uuid4())
```